src/state/slices/stakingDataSlice/selectors.ts

```diff
@@ -0,0 +1,190 @@
+import { createSelector } from '@reduxjs/toolkit'
+import { CAIP10, CAIP19 } from '@shapeshiftoss/caip'
+import { bnOrZero } from '@shapeshiftoss/chain-adapters'
+import { ValidatorReward } from '@shapeshiftoss/types/dist/chain-adapters/cosmos'
+import { ReduxState } from 'state/reducer'
+import { createDeepEqualOutputSelector } from 'state/selector-utils'
+
+export const ASSET_ID_TO_DENOM: { [k: CAIP19]: string } = {
+  'cosmos:cosmoshub-4/slip44:118': 'uatom'
+}
+
+export const DENOM_TO_ASSET_ID: { [k: string]: CAIP19 } = {
+  uatom: 'cosmos:cosmoshub-4/slip44:118'
```

Comment:
Is it expected that we don't yet use this anywhere?

Reply:
Yeah it's there just to mirror the first for reference, both will go away eventually so I left it there in case we need it/for visual reference.

Reply:
FYI we're going to have the denom directly in the `Asset` type, coming from asset service so none of these two will be needed